.neutrinorc.js

```diff
@@ -1,9 +1,11 @@
+const eslint = require('@neutrinojs/eslint');
+const react = require('@neutrinojs/react');
+const copy = require('@neutrinojs/copy');
+const jest = require('@neutrinojs/jest');
+
 // This is the configuration file for Neutrino, which configures webpack and Jest:
 // https://neutrinojs.org
 
-// `use strict` is still necessary here since this file is not treated as a module.
-'use strict'; // eslint-disable-line strict, lines-around-directive
```

Comment:
The reasons for including this in the file still exist (if the migrate tool removed it, it's a bug)

Reply:
Oh, that was a mistake. The lines-around-directive was hit while linting: `Unused eslint-disable directive (no problems were reported from 'lines-around-directive')`

Reply:
Adding it back in, it's not hitting that error now... odd